peer/pendingheap/heap.go

```diff
@@ -56,7 +56,7 @@ func (ph *pendingHeap) Choose(ctx context.Context, req *transport.Request) peer.
 	return ps.peer
 }
 
-func (ph *pendingHeap) Add(p peer.StatusPeer) peer.Subscriber {
+func (ph *pendingHeap) Add(p peer.StatusPeer, _ peer.Identifier) peer.Subscriber {
```

Comment:
Can we add a compile-time check for the `Implementation` interface here so that we can readily see what `pendingHeap` is implementing?

```
var _ peerlist.Implementation = (*pendingHeap)(nil)
```

Reply:
✔